### PR TITLE
providing documentation to UAA feature

### PR DESCRIPTION
**Pull request by xetys**
this PR consits of a page, linked at the security page deeply explaining my oauth2 design on microservice architecture and its application on the UAA featue series

ready to any review

### CODE REVIEW

File: pages/security.html [hunk 1]
```diff
@@ -77,3 +77,11 @@
 <p>
     The secret key should be configured in the <code>application.yml</code> file, as the <code>jhipster.security.authentication.jwt.secret</code> property.
 </p>
+
+<h2>Securing Microservices with OAuth2 and UAA-Service</h2>
+<p>
+  When it comes to securing microservices, there are several claims to a security solution to satisfy.
+  So in scale, using OAuth2 protocolls perfectly fit all these needs.
```
Comment (jdubois) [inline pages/security.html]: protocols (only one "l")

File: pages/using-uaa.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+
+## Summary
+
+1. [security claims of microservice architecture](#claims)
+2. [understanding OAuth2 in this context](#oauth2)
+3. [using JHipster UAA](#jhipster-uaa)
+  * basic setup
+  * understanding the components
+  * commong mistakes
```
Comment (jdubois) [inline pages/using-uaa.md]: common (no "g")

**Comment by xetys** [comment]
where this uaa doc page should be linked? Did I picked the right place our you would prefere it somewhere in "Doing Microservices"?

**Comment by xetys** [comment]
so, here is my early version of the docs. be free to criticize it, since it should be a good reference for all java hipsters :)

i am back to the beach for now, 3 PRs pending is a good rate while having vacation :laughing:

**Comment by jdubois** [comment]
Thanks @xetys 

- Yes I will put this documentation in the "doing microservices" part, not in the security part (that's what we do for all microservices-related tech)
- I will review everything, and put a few "beta" tags on it, as I don't think it's ready for prod, but don't hesitate to tell me if you think it should be OK

**Comment by xetys** [comment]
i can link this to that page, no problem

of cause, everything what is not ready for prod, should be noted as beta...we are talking about security, and this is not a topic where things should be hesitated at all..

**Comment by jdubois** [comment]
@xetys I'm refactoring and rewording stuff - don't worry, you'll have the diff and we can still change it later anyway

**Pull request merged**